### PR TITLE
accumulate: move important hint to instructions

### PR DESCRIPTION
**Pull request by senekor**

### CODE REVIEW

**Comment by senekor** [comment]
[forum discussion](https://forum.exercism.org/t/accumulate-start/14074)

File: exercises/practice/accumulate/.docs/instructions.append.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Instructions append
+
+## Workflow
+
+The tests for this exercise will cause compile time errors,
```
Comment (BNAndras) [inline exercises/practice/accumulate/.docs/instructions.append.md:5]:
```suggestion
The tests for this exercise will cause compile time errors.
```

**Review by BNAndras — CHANGES_REQUESTED** [review]
Looks good. One minor typo.

File: exercises/practice/accumulate/.docs/instructions.append.md (hunk 1 above)
Comment (senekor) [reply exercises/practice/accumulate/.docs/instructions.append.md]: The next line is actually part of the sentence, I put it all on one line.

**Review by BNAndras — APPROVED** [review]

**Pull request merged**